README.md

```diff
@@ -14,7 +14,7 @@ This action currently supports these versions of MRI, JRuby and TruffleRuby:
 
 | Interpreter | Versions |
 | ----------- | -------- |
-| Ruby | 2.2, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
+| Ruby | 2.1.9, 2.2.10, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
```

Comment:
For 2.2 it's 2.2.6 on Windows and 2.2.10 otherwise (see below), unfortunately.
So I guess `2.2` in the description is best.

Reply:
Hrmpf... You are right of course :)